src/components/symptom-shark/container/LogEntryList/LogEntryList.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from "react";
+import { add } from "date-fns";
+import MyDataHelps from "@careevolution/mydatahelps-js"
+import { previewConfiguration, previewLogEntry } from "../LogToday/LogToday.previewData";
+import symptomSharkData, { DailyLogEntry, SymptomSharkConfiguration } from "../../helpers/symptom-shark-data";
+import getDayKey from "../../../../helpers/get-day-key";
+import { SymptomSharkLogEntry } from "../../presentational";
+import OnVisibleTrigger from "../../../presentational/OnVisibleTrigger";
+import { LoadingIndicator } from "../../../presentational";
+import language from "../../../../helpers/language";
+
+export interface SymptomSharkLogEntryListProps {
+    onDaySelected(d: Date): void;
+    previewState?: "default";
+}
```

Comment:
Do we need to add `innerRef` to any of these?

Reply:
Ah yep, probably do!